warehouse/ingest-core/src/main/java/datawave/util/flag/FlagMaker.java

```diff
@@ -266,13 +266,18 @@ void loadFilesForDistributor(FlagDataTypeConfig fc, FileSystem fs) throws IOExce
                 } else {
                     try {
                         this.fd.addInputFile(new InputFile(folder, status, this.fmc.getBaseHDFSDir(), this.fmc.isUseFolderTimestamp()));
+                        logFileInfo(fc, status);
                     } catch (UnusableFileException e) {
                         log.warn("Skipping unusable file " + status.getPath(), e);
                     }
                 }
             }
         }
     }
+
+    protected void logFileInfo(FlagDataTypeConfig fc, FileStatus status) {
+        log.info("File " + fc.getDataName() + " : " + status);
```

Comment:
This should probably be:
```
log.info("File {} : {}", fc.getDataName(), status);
```
So that we don't have to compute the combined string if we're not logging to the info level.

Reply:
done.